src/Symfony/Component/VarDumper/Caster/DateCaster.php

```diff
@@ -27,8 +27,8 @@ class DateCaster
     public static function castDateTime(\DateTimeInterface $d, array $a, Stub $stub, bool $isNested, int $filter)
     {
         $prefix = Caster::PREFIX_VIRTUAL;
-        $location = $d->getTimezone()->getLocation();
-        $fromNow = (new \DateTime())->diff($d);
+        $location = $d->getTimezone() ? $d->getTimezone()->getLocation() : null;
+        $fromNow = $d->getTimezone() ? (new \DateTime())->diff($d) : new \DateInterval('PT0S');
```

Comment:
This is wrong. Having no timezone does not imply an offset of zero.

Reply:
You are right, I have changed it.